_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,308 @@
+<div class="example-set">
+    <div class="list">
+        <div class="is-current item item-complex">
+            <h4 class="item-hd">Home</h4>
+            <p class="item-copy">You know, where the heart is</p>
+        </div>
+        <div class="item item-complex">
+            <span class="item-hd">Darnassea</span>
+            <p class="item-copy">Designed on top of an ocean, the town of Darnassea is home to dark elves lead by Colonel Alduin.
+This town wasn't built by an ocean by accident, as it has ancient, lost technologies, which is of great importance to the people of Darnassea and its success.</p>
+        </div>
+        <div class="item item-complex">
+            <h5 class="item-hd hd-2">Complex Item Heading</h5>
+            <p class="item-copy copy-large">Darkwell has a troubled economy, which is mainly supported by jewelcrafting, fishing and fletching. But their biggest strengths are intricate jewelcrafting and rare herbalism. However, Darkwell lacks people skilled in farming.</p>
+        </div>
+        <div class="item item-complex">
+            <h2 class="item-hd hd-5">Darkwell</h2>
+            <p class="item-copy copy-meta">Raised inside a geyser field, the port of Darkwell is home to vikings lead by Mrs. Chilson. This port wasn't built by a geyser field by accident, as it has rare resources, which is of great importance to the people of Darkwell and its success.</p>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Grouped Lists</h3>
+<div class="example-set">
+    <ul class="list list-grouped">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item"><a href="http://ux.edx.org">Bananas</a></li>
+        <li class="item">Blackberries</li>
+    </ul>
+    
+    <div class="list list-grouped">
+        <div class="item item-complex">
+            <span class="item-hd">Darnassea</span>
+            <p class="item-copy">Designed on top of an ocean, the town of Darnassea is home to dark elves lead by Colonel Alduin.
+This town wasn't built by an ocean by accident, as it has ancient, lost technologies, which is of great importance to the people of Darnassea and its success</p>
+        </div>
+        <div class="item item-complex">
+            <h5 class="item-hd hd-2">Darkwell</h5>
+            <p class="item-copy copy-large">Darkwell has a troubled economy, which is mainly supported by jewelcrafting, fishing and fletching. But their biggest strengths are intricate jewelcrafting and rare herbalism. However, Darkwell lacks people skilled in farming.</p>
+        </div>
+    </div>
+
+    <ul class="list list-grouped">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item item-complex has-block-link">
```

Comment:
Hmm, let's chat about this with the local demo in front of us - I'm not sure I'm following.